node/resolvers/saveMasterdata.ts

```diff
@@ -0,0 +1,33 @@
+export async function saveMasterdata(
+  _: unknown,
+  saveValues: SaveValues,
+  ctx: Context
+) {
+  if (saveValues.saveData.name.length === 0) {
+    throw new Error('É necessário enviar um name')
+  }
+
+  if (saveValues.saveData.content.length === 0) {
+    throw new Error('É necessário enviar um content')
+  }
+
+  if (saveValues.saveData.operator.length === 0) {
+    throw new Error('É necessário enviar um operator')
+  }
+
+  if (saveValues.saveData.simpleStatements.length === 0) {
+    throw new Error('É necessário enviar um simpleStatements')
+  }
+
+  if (saveValues.saveData.type.length === 0) {
+    throw new Error('É necessário enviar um type')
+  }
```

Comment:
Just a suggestion and a curiosity to. You can use destructuring to improve your code here, to avoid use repetitive objects.

Example:

```suggestion

  const { name, content, operator, simpleStatements, type } = saveValues.saveData

  if (name.length === 0) {
    throw new Error('É necessário enviar um name')
  }

  if (content.length === 0) {
    throw new Error('É necessário enviar um content')
  }

  if (operator.length === 0) {
    throw new Error('É necessário enviar um operator')
  }

  if (simpleStatements.length === 0) {
    throw new Error('É necessário enviar um simpleStatements')
  }

  if (type.length === 0) {
    throw new Error('É necessário enviar um type')
  }
```